tools/export/makefile/__init__.py

```diff
@@ -102,6 +105,38 @@ def generate(self):
         else:
             raise NotSupportedException("This make tool is in development")
 
+    @staticmethod
+    def build(project_name, log_name="build_log.txt", cleanup=True):
+        """ Build Make project """
+        # > Make -C [project directory] -j
+        cmd = ["make", "-j"]
+        p = Popen(cmd, stdout=PIPE, stderr=PIPE)
+        ret = p.communicate()
+        out, err = ret[0], ret[1]
+        ret_code = p.returncode
+        with open(log_name, 'w+') as f:
+            f.write("=" * 10 + "OUT" + "=" * 10 + "\n")
+            f.write(out)
+            f.write("=" * 10 + "ERR" + "=" * 10 + "\n")
+            f.write(err)
+            if ret_code == 0:
+                f.write("SUCCESS")
+            else:
+                f.write("FAILURE")
+        with open(log_name, 'r') as f:
+            print "\n".join(f.readlines())
+        sys.stdout.flush()
+
+        if cleanup:
+            remove("Makefile")
+            remove(log_name)
+            if exists('.build'):
+                shutil.rmtree('.build')
+        if ret_code != 0:
+            # Seems like something went wrong.
+            return -1
```

Comment:
Is this documented, now that we have return values, that -1 is an error, 0 succcess ? I can't find it in this changeset. Previously ``build`` function raised an exception, no return value.